packages/gestalt-codemods/0.108.0-0.109.0/__testfixtures__/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "prettier/prettier": 0,
```

Comment:
Had to disable prettier in the test fixture because of
```
<Text size="xl" weight={isCurrentPage ? "bold" : "normal"}>
```
should instead be
```
<Text size="xl" weight={isCurrentPage ? 'bold' : 'normal'}>
```

However, when calling jscodeshift with `.toSource({quote: 'single'})`, it converts all quotes instead of only the modified ones.